homeassistant/components/google_tasks/__init__.py

```diff
@@ -17,7 +17,10 @@
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up Google Tasks from a config entry."""
-    hass.data.setdefault(DOMAIN, {})
+
+    if not entry.unique_id:
```

Comment:
I am not sure its worth forcing a reauth on everyone

Reply:
Hmm, on one side I think it's a good idea to have everyone set the unique id of the config entry, but now that I think of it, it's not needed for it to work.